src/containers/Cluster/Cluster.tsx

```diff
@@ -1,22 +1,23 @@
+import {useRouteMatch} from 'react-router';
 import cn from 'bem-cn-lite';
-//@ts-ignore
+
+import type {AdditionalVersionsProps} from '../../types/additionalProps';
+import routes, {CLUSTER_PAGES} from '../../routes';
+
+import {ClusterInfo} from '../ClusterInfo/ClusterInfo';
 import Tenants from '../Tenants/Tenants';
-//@ts-ignore
 import {Nodes} from '../Nodes/Nodes';
-//@ts-ignore
 import Storage from '../Storage/Storage';
-import routes, {CLUSTER_PAGES} from '../../routes';
 
 import './Cluster.scss';
-import {useRouteMatch} from 'react-router';
-import {ClusterInfo} from '../ClusterInfo/ClusterInfo';
 
 const b = cn('cluster');
 
 interface ClusterProps {
     additionalClusterInfo?: any;
     additionalTenantsInfo?: any;
     additionalNodesInfo?: any;
+    additionalVersionsProps?: AdditionalVersionsProps;
```

Comment:
The naming is intentional, I don't like `...info`. I want to alter it for other `additional...` as well. However, I'm not sure, that it is the best, so if you have better ideas, you are welcome